### PR TITLE
Use the new version file to check for updates

### LINKED ISSUE #2305: Use the new version file format for checks
After https://github.com/pi-hole/pi-hole/pull/4875 the versions and branches will be in a single file.

The PHP code needs to reflect this change.

### PR DESCRIPTION
**Pull request by rdwebdesign**
- **What does this PR aim to accomplish?:**

This PR uses the new file (created by https://github.com/pi-hole/pi-hole/pull/4875) to check for updates.
It also adds a specific check for new docker tags.

- **How does this PR accomplish the above?:**

Replacing the old files and using the new unified file to look for local versions/branches/new versions.
Added code to show "docker" updates.

- **What documentation changes (if any) are needed to support this PR?:**

none

### Warning
Only merge after https://github.com/pi-hole/pi-hole/pull/4875 is merged.

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
- [x] I have read the above and my PR is ready for review. _Check this box to confirm_

### CODE REVIEW

**Comment by rdwebdesign** [comment]
image

File: scripts/pi-hole/php/footer.php [hunk 1]
```diff
@@ -72,20 +57,20 @@
...
                     <li><strong>Web Interface</strong> <?php echo $webVersionStr; ?></li>
                 </ul>
                 <?php } else { ?>
                 <ul class="list-inline">
-                    <?php if ($dockerTag) { ?>
+                    <?php if ($docker_current) { ?>
                     <li>
                         <strong>Docker Tag</strong>
-                        <a href="<?php echo $dockerReleasesUrl.'/'.$dockerTag; ?>" rel="noopener" target="_blank"><?php echo $dockerTag; ?></a>
+                        <a href="<?php echo $dockerReleasesUrl.'/'.$docker_current; ?>" rel="noopener" target="_blank"><?php echo $docker_current; ?></a>
```
Comment (PromoFaux) [inline scripts/pi-hole/php/footer.php]:
```suggestion
                        <a href="<?php echo $dockerReleasesUrl.'/'.$docker_current; ?>" rel="noopener" target="_blank"><?php echo $docker_current; ?></a>
                        <?php if ($docker_update) { ?> &middot; <a class="lookatme" lookatme-text="Update available!" href="<?php echo $dockerReleasesUrl.'/latest'; ?>" rel="noopener" target="_blank">Update available!</a><?php } ?>
```

Comment (PromoFaux) [reply scripts/pi-hole/php/footer.php]: And actually "while you're at it".. you may as well add in the suppression that I was planning to add in my PR

Comment (rdwebdesign) [reply scripts/pi-hole/php/footer.php]: I think I did it in the `update_checker.php` code.

Here:
https://github.com/pi-hole/AdminLTE/blob/5ce401a07ee0072130be0f6c061e4efa62bffc3e/scripts/pi-hole/php/update_checker.php#L71-L81

If it's a docker container, I'm not even checking each component version individually.

Comment (PromoFaux) [reply scripts/pi-hole/php/footer.php]: Ah, right ok - yep I see the logic there. In which case I think my PR is null and void?

Comment (PromoFaux) [reply scripts/pi-hole/php/footer.php]: But I think you should still add the `lookatme` thingy 
image

Comment (rdwebdesign) [reply scripts/pi-hole/php/footer.php]: I will make the changes here.

After that, your PR will be unnecessary.

**Review by PromoFaux — APPROVED** [review]

**Comment by yubiuser** [comment]
Logspam ahead:

```
2022-08-28 20:36:16: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined index: GITHUB_DOCKER_VERSION in /var/www/html/admin/scripts/pi-hole/php/update_checker.php on line 68
2022-08-28 20:36:16: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined variable: docker_update in /var/www/html/admin/scripts/pi-hole/php/footer.php on line 93
2022-08-28 20:36:41: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined index: GITHUB_DOCKER_VERSION in /var/www/html/admin/scripts/pi-hole/php/update_checker.php on line 68
2022-08-28 20:36:41: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined variable: docker_update in /var/www/html/admin/scripts/pi-hole/php/footer.php on line 93
2022-08-28 20:39:04: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined index: GITHUB_DOCKER_VERSION in /var/www/html/admin/scripts/pi-hole/php/update_checker.php on line 68
2022-08-28 20:39:04: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined variable: docker_update in /var/www/html/admin/scripts/pi-hole/php/footer.php on line 93
2022-08-28 20:39:40: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined index: GITHUB_DOCKER_VERSION in /var/www/html/admin/scripts/pi-hole/php/update_checker.php on line 68
2022-08-28 20:39:40: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined variable: docker_update in /var/www/html/admin/scripts/pi-hole/php/footer.php on line 93
2022-08-28 20:41:09: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined index: GITHUB_DOCKER_VERSION in /var/www/html/admin/scripts/pi-hole/php/update_checker.php on line 68
2022-08-28 20:41:09: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined variable: docker_update in /var/www/html/admin/scripts/pi-hole/php/footer.php on line 93
```

**Pull request merged**